rskj-core/src/main/java/co/rsk/rpc/netty/JsonRpcWeb3ServerHandler.java

```diff
@@ -86,6 +86,12 @@ private ByteBuf buildErrorContent(int errorCode, String errorMessage) throws Jso
         errorProperties.put("code", jsonNodeFactory.numberNode(errorCode));
         errorProperties.put("message", jsonNodeFactory.textNode(errorMessage));
         JsonNode error = jsonNodeFactory.objectNode().set("error", jsonNodeFactory.objectNode().setAll(errorProperties));
-        return Unpooled.wrappedBuffer(mapper.writeValueAsBytes(mapper.treeToValue(error, Object.class)));
+        Object object = mapper.treeToValue(error, Object.class);
+
+        if (object == null) {
+            throw new NullPointerException();
```

Comment:
I know this is unlikely to be reached, but I would rather return sth like the following instead of throwing, otherwise the other party will not receive anything as response from the node (`ctx.fireChannelRead...` will be skipped).

`return Unpooled.wrappedBuffer(mapper.writeValueAsBytes("unknown error"));`

Reply:
Not necessary anymore with the latest changes